SparkPost/2web2ui
# SA-1517: Precision sticks only when explicitly set or non date update

### What Changed
 - Precision only sticks if explicitly set and it's hibana. Otherwise, falls back on preexisting logic for precision

### How To Test
 - Go to non hibana summary report
 - Go to 24 hour range
   - Precision should be in hours
- Go to 7 days
  -Precision should be in days
 - Go to 30 days
  - Precision should be in days
 - Go to 7 days
  -Precision should be in days

### To Do
- [ ] Double check if rounding is working correctly
